doc/userguide/rules/dhcp-keywords.rst

```diff
@@ -19,3 +19,41 @@ The version can be matched exactly, or compared using the _op_ setting::
 Signature example::
 
  alert dhcp any any -> any any (msg:"small DHCP lease time (<3)"; dhcp.leasetime:<3; sid:1; rev:1;)
+
+dhcp.rebinding_time
+-------------------
+
+DHCP rebinding time (integer).
+
+Syntax::
+
+ dhcp.rebinding_time:[op]<number>
+
+The version can be matched exactly, or compared using the _op_ setting::
```

Comment:
I believe that here and in the other places where `version` is mentioned here, we are actually talking about the specific keywords described, right?

Reply:
Thanks :-)